src/_rules/shadow.js

```diff
@@ -1,6 +1,12 @@
+const defaultColor = 'rgba(0, 0, 0 , .2)';
```

Comment:
Just a minor nitpick

```suggestion
const defaultColor = 'rgba(0, 0, 0, .2)';
```

Reply:
Ah, thanks! 🦅